openedx_webhooks/info.py

```diff
@@ -48,6 +48,18 @@ def get_orgs(key):
     orgs = get_orgs_file()
     return set(o for o, info in orgs.items() if info.get(key, False))
 
+def get_person_time_of_pull_request(person, created_at):
+    """
+    Return person data structure as of time of pull request
+    """
+    for before_date in sorted(person.get("before", {})):
+        if created_at.date() <= before_date:
+            before_person = person.get("before")[before_date]
+            update_person = person.copy()
+            update_person.update(before_person)
```

Comment:
I'd need to look at the data again to be sure about this being enough.  If the current person has extra values that aren't mentioned in the before data, then the updated person will have a mix of old and new.  I'm not sure if that happens, or if it matters?

Reply:
The person will have a mix of old and new values, but I don't believe that actually matters. We are only checking the "agreement", "institution" and "expires_on" keys as the other information, other than the Github username, is currently irrelevant (I think).